### ISSUE TITLE
Details regarding training speed

### ISSUE THREAD

**Issue by khushigupta** [rwightman/efficientdet-pytorch#39]
Can you comment on how long it takes you to train a `tf_efficientdet_d0` from scratch on COCO using pretrained imagenet weights?

I'm training with the following command on 2 RTX 6000s and it's taken ~18.5 hours to reach 41 epochs.
`./distributed_train.sh 2 /mscoco --model tf_efficientdet_d0 -b 48 --amp --lr .06 --warmup-epochs 5 --sync-bn --opt fusedmomentum --fill-color mean --model-ema`

**Comment by rwightman** [comment]
I get approx 95-100img/s on D0 with 2 Titan RTX or 4 1080Ti, about 4 days for 300 epochs. 

BTW you need to scale the learning rate linearly with batch size so your command has too small of a learning rate as your effective batch size is 96.
